pkg/kubelet/dockertools/manager.go

```diff
@@ -1354,6 +1354,10 @@ func (dm *DockerManager) killContainer(containerID kubecontainer.ContainerID, co
 	if gracePeriod < minimumGracePeriodInSeconds {
 		gracePeriod = minimumGracePeriodInSeconds
 	}
+	if gracePeriodOverride != nil {
```

Comment:
Add a comment here about why gracePeriodOverride is allowed to go below minimum (because this is a hard kill) and ONLY hard kill paths are allowed to do this.  I.e. no one else in the entire Kubelet should ever set this value less than minimumGracePeriodInSeconds, otherwise they can and will corrupt user data.